service/src/main/java/uk/gov/hmcts/reform/fpl/utils/PeopleInCaseHelper.java

```diff
@@ -16,6 +18,17 @@ private PeopleInCaseHelper() {
         // NO-OP
     }
 
+    public static String getFirstApplicantName(List<Element<Applicant>> applicants) {
```

Comment:
as discussed. we will need test(s) for this

Reply:
done